sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/library/build.sbt

```diff
@@ -6,9 +6,7 @@ scalaVersion := "2.11.8"
 
 npmDependencies in Compile += "uuid" -> "3.1.0"
 
-//#relevant-settings
-webpackConfigFile := Some(baseDirectory.value / "webpack.config.js")
-//#relevant-settings
+bundlingMode := BundlingMode.LibraryAndApplication()
```

Comment:
You should keep the `//#relevant-settings` so that you can use this part of the file from the documentation.